tools/rtpsrelay/RelayHandler.cpp

```diff
@@ -291,6 +291,11 @@ CORBA::ULong VerticalHandler::process_message(const ACE_INET_Addr& remote_addres
   {
     GuidAddrSet::Proxy proxy(guid_addr_set_);
     proxy.process_expirations(now);
+    if (!proxy.check_address(remote_address)) {
+      HANDLER_WARNING((LM_WARNING, ACE_TEXT("(%P|%t) WARNING: VerticalHandler::process_message %C dropping message from rejected address %C\n"),
```

Comment:
Remove HANDLER_WARNING.

Reply:
Completely remove, or different guard? Assuming you mean remove completely, did we want to record the dropped packet somehow? I can't remember what we decided, if we wanted to collect a statistic for rejected messages, or maybe just lump them in with a different statistic (ignored)...

Reply:
Guard is correct.  The log message could flood so just record the message/bytes in the `ignore_*` fields of the handler.